server/resources/UserResources.py

```diff
@@ -39,6 +43,10 @@ def post(self):
             return {'message': 'Something went wrong'}, 500
 
 class UserLogin(Resource):
+<<<<<<< HEAD
+    #@validate_args("email", "password")
+=======
+>>>>>>> origin
```

Comment:
probably need to address these merge conflicts

Reply:
Oops, good catch